OpenRA.Mods.Common/UpdateRules/UpdateUtils.cs

```diff
@@ -272,6 +272,22 @@ public static void RemoveNode(this MiniYamlNode node, MiniYamlNode toRemove)
 			node.Value.Nodes.Remove(toRemove);
 		}
 
+		public static void MoveNode(this MiniYamlNode node, MiniYamlNode fromNode, MiniYamlNode toNode)
+		{
+			toNode.Value.Nodes.Add(node);
+			fromNode.Value.Nodes.Remove(node);
+		}
+
+		public static void MoveAndRenameNode(this MiniYamlNode node, MiniYamlNode fromNode, MiniYamlNode toNode, string newKey, bool preserveSuffix = true)
+		{
+			if (preserveSuffix)
+				node.RenameKeyPreservingSuffix(newKey);
+			else
+				node.Key = newKey;
+
+			node.MoveNode(fromNode, toNode);
+		}
+
 		/// <summary>Removes children with keys equal to [match] or [match]@[arbitrary suffix]</summary>
 		public static int RemoveNodes(this MiniYamlNode node, string match)
```

Comment:
We probably also want a `bool matchSuffix=true` parameter on these too for consistency.  Thankfully none of our the rules will need to change to go with this.